content/config/daemon/proxy.md

```diff
@@ -37,39 +52,35 @@ its start-up environment to configure HTTP or HTTPS proxy behavior:
 - `NO_PROXY`
 - `no_proxy`
 
-In Docker Engine version 23.0 and later versions, you may also configure proxy
-behavior for the daemon in the [`daemon.json` file](./index.md#configure-the-docker-daemon):
+Set any of these variables and restart the daemon for the proxy configuration
+to take effect:
 
-```json
-{
-  "proxies": {
-    "http-proxy": "http://proxy.example.com:3128",
-    "https-proxy": "https://proxy.example.com:3129",
-    "no-proxy": "*.test.example.com,.example.org,127.0.0.0/8"
-  }
-}
+```console
+$ export HTTP_PROXY=http://proxy.example.com:3128
+$ export HTTPS_PROXY=https://proxy.example.com:3129
+$ export NO_PROXY=localhost,127.0.0.1,docker-registry.example.com,.corp
+$ sudo -E dockerd
```

Comment:
IMHO it's a bit weird to use `sudo -E dockerd` here. I wouldn't expect anyone to start dockerd that way. Maybe use a placeholder comment instead, or even nothing at all.